app/components/view-question-data-text/template.njk

```diff
@@ -20,17 +21,16 @@
   <h3>Generated HTML</h3>
   <pre><code>
     {{- "
-      <div data-test-id='view-question-data-text-some-id'>
-        <label class='nhsuk-label'>
-          Some data
-        </label>
+      <div data-test-id='view-question-data-text-some-id' class='bc-c-data-text another-class'>
+        Some data
       </div>
     " }}
   </code></pre>
```

Comment:
Remove this.

Reply:
what?

Reply:
the generated html block in the template